app/controllers/application_controller.rb

```diff
@@ -1,6 +1,13 @@
 # rubocop:disable all
 class ApplicationController < ActionController::Base
-    def current_user
-        User.first
-      end
+    protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
```

Comment:
- [ ] I am able to visit different routes without signing in. Kindly fix that. Currently adding the following helper method fixes that.

  ```
  before_action :authenticate_admin_user!
  ```
  But after fixing the resulting issue with User model, you need to add this helper to this controller to ensure only signed in users can visit certain routes.

  ```
  before_action :authenticate_user!
  ```